index.js

```diff
@@ -1 +1,44 @@
-document.write('<script src="js/index.js"></script>');
+// Get the elements for the pop-up menu and the union-bar
+const popUpSection = document.querySelector('.pop-up-section');
+const unionBar = document.querySelector('.union-bar');
+const crossBtn = document.querySelector('.cross-btn');
+const popUpItems = document.querySelectorAll('.pop-menu-item');
+
+// Function to toggle the visibility of the pop-up menu
+function togglePopUp() {
+  popUpSection.classList.toggle('show');
+}
+
+// Add a click event listener to the union-bar
+unionBar.addEventListener('click', togglePopUp);
+
+// Function to hide the pop-up menu when a menu item is clicked
+function hidePopUp() {
+  popUpSection.classList.remove('show');
+}
+
+// Add a click event listener to each pop-up menu item
+popUpItems.forEach((item) => {
+  item.addEventListener('click', hidePopUp);
+});
+
+// Function to hide the pop-up menu when the cross button is clicked
+function hidePopUpWithCross() {
+  popUpSection.classList.remove('show');
+}
+
+// Add a click event listener to the cross button
+crossBtn.addEventListener('click', hidePopUpWithCross);
+// Scroll smoothly to the section when a menu item is clicked
+function smoothScrollToSection(e) {
+  e.preventDefault();
+  const target = document.querySelector(e.target.getAttribute('href'));
+  if (target) {
+    const yOffset = -50; // Optional offset if you have a sticky header
+    const y = target.getBoundingClientRect().top + window.pageYOffset + yOffset;
+    window.scrollTo({ top: y, behavior: 'smooth' });
+  }
+  hidePopUp();
+}
```

Comment:
- [ ] Great job working on the menu so far but I think there is an issue since when I open your portfolio on the mobile version I could see the menu open but it should open when I press the hamburger button, and when I try to close it, it doesn't close so kindly fix that so when I press on the hamburger it open and when I press on X button it closes 💯⭐🔥

![image](https://github.com/lucy-sees/Webforlio/assets/58553711/dc8c8481-9f3a-45f5-b04b-6333d09506f7)



https://github.com/lucy-sees/Webforlio/assets/58553711/c9286d07-5585-49a6-8164-76ec8820aee5